examples/process-persistence/README.md

```diff
@@ -0,0 +1,373 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+-->
+
+# Example :: Process Persistence
+
+This example depicts the usage of persistence configuration for various supported databases like H2, Postgresql and
+MS SQL. This application can be run in dev and container modes.
+
+This example also showcases a basic implementation of a **Hiring** Process that drives a _Candidate_ through different
+interviews until they get hired. It features simple User Task orchestration including the use of DMN decisions to
+generate the candidate's offer and timers to skip interviews.
+
+---
+
```

Comment:
@josedee something  I'm missing in the readme is a having a bit more info regarding the persistence config, I think that instead of explaining the process (the readme could simply say that the example is based on the `process-compact-architecture` and just refer to it for details).

I think, it would be good to have a chapter talking about config, which deps are needed for each databases, settings in application.properties etc.